run

```diff
@@ -225,6 +227,9 @@ parse_options(){
         elif [ "$1" == "-K" -o "$1" == "--keep" ]; then
             [ -t 1 ] && KOPT+=("ktest.keep")
             shift
+        elif [ "$1" == "--shared" ]; then
```

Comment:
If wonder if we should make it an attribute of a DISKSPEC rather than global setting for all disks.